### DISCUSSION TITLE
MIXCR OUTPUT TABLE_CLONOTYPE AND SHMT

### DISCUSSION THREAD

**Discussion by singhd6** [milaboratory/mixcr#1852] [Q&A] [answered]
Question1. I analyzed my data with generic amplicon code and i generated the clonotype table and SHMT table, however i do not have unique UMI for form Fastq files, Now I am thinking about selecting the most relevant CDR3 sequences based on the mutation frequency, but I can not calculate the mutation frequency through the mutate function in R without having unique UMIs. However in mixcr we have nMutation rate, can we consider the nMutation rate as mutation frequency?
Question2. In SHMT table - we have the treeId, nodeID, can we consider the maximum number of nodeID as the nodes or branches of B cells?
[mixcr_genericamplicon.docx](https://github.com/user-attachments/files/17742480/mixcr_genericamplicon.docx)

**Comment by mizraelson** [comment, marked as answer]
- You do not need UMIs to calculate mutation frequency, as it is unrelated. You can use readCount instead. How do you define the most relevant CDR3? Is it the most abundant? If so, you should use readCount.

- The mutation rate is calculated as the number of mutations in the region divided by the length of the region.

- There is a column labeled numberOfClonesInTree that reflects the size of the tree. Additionally, the more unique node IDs you observe, the larger the tree.”
